dataAccessObjects.py

```diff
@@ -323,6 +339,9 @@ def getTextMeta(self,textId):
     def generateNewId():
         return super().generateNewId()
 
+    def updateEditedData(self,data):
```

Comment:
You can delete this function, because the derived class can use the method of parent class